### PR TITLE
SASS-3536 - Create Jobseeker's Allowance 'Add missing claim' page.

### PR DESCRIPTION
**Pull request by georgievh**
### Description

Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-3536](https://jira.tools.tax.service.gov.uk/browse/SASS-3536)

##### Before Reviewing

- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing

- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing

- [x]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request?
  i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR

- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised

- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

File: test/utils/InYearUtilSpec.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  with MockFactory {
+
+  private val year2022: Int = 2022
+  private val month4: Int = 4
+  private val day5: Int = 5
+  private val day6: Int = 6
+  private val hour23: Int = 23
+  private val minute59: Int = 59
+
+  private def newDate(year: Int, month: Int, day: Int): LocalDateTime = LocalDateTime.of(year, month, day, 0, 0)
```
Comment (rtna-hmrc) [inline test/utils/InYearUtilSpec.scala:34]: It seems it is only used in line 45. Maybe we can just inline it?

**Comment by rtna-hmrc** [comment]
image
The play-frontend-hmrc lib can be upgraded

File: test/utils/InYearUtilSpec.scala (hunk 1 above)
Comment (georgievh) [reply test/utils/InYearUtilSpec.scala]: Done

**Comment by georgievh** [comment]
Library updated.

**Review by rtna-hmrc — APPROVED** [review]

**Pull request merged**